### PR TITLE
Conform message center input to dynamic type font

### PR DESCRIPTION
**Pull request by rasmustautsglia**
Conform Welcome Screen to be ADA compliant.

MOB-2489

### CODE REVIEW

**Review by EgorovEI — APPROVED** [review]

**Review by gersonnoboa — COMMENTED** [review]
Good but the commit doesn't have description or ticket number.

File: GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeView.swift [hunk 1]
```diff
@@ -543,11 +543,13 @@ extension SecureConversations.WelcomeView {
 
         lazy var textView = UITextView().makeView { textView in
             textView.backgroundColor = .clear
+            textView.adjustsFontForContentSizeCategory = true
```
Comment (igorkravchenko) [inline GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeView.swift:546]: I think there are two things here to think about in terms of lacking consistency:
1. We should use dedicated free function `setFontScalingEnabled` to handle specific cases like [this](https://github.com/salemove/ios-sdk-widgets/blob/52fb8e3b24eec85a98f9f7233e508b35d8d148a0/GliaWidgets/Sources/FontScaling/FontScaling.swift#L81) for example.
2. This should be controlled through dedicated style `isFontScalingEnabled`.

File: GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeView.swift [hunk 2]
```diff
@@ -543,11 +543,13 @@ extension SecureConversations.WelcomeView {
...
             textView.delegate = self
         }
 
         lazy var placeholderLabel = UILabel().makeView { label in
             label.numberOfLines = 0
+            label.adjustsFontForContentSizeCategory = true
```
Comment (igorkravchenko) [inline GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeView.swift:552]: [Same](https://github.com/salemove/ios-sdk-widgets/pull/744/files#r1311285487).

Comment (rasmustautsglia) [reply GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeView.swift]: fixed [ab8bcde](https://github.com/salemove/ios-sdk-widgets/pull/744/commits/ab8bcde359e057a2f05f6c4d32460e8d7f198011)

File: GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeView.swift (hunk 1 above)
Comment (rasmustautsglia) [reply GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeView.swift]: fixed [ab8bcde](https://github.com/salemove/ios-sdk-widgets/pull/744/commits/ab8bcde359e057a2f05f6c4d32460e8d7f198011)

**Comment by rasmustautsglia** [comment]
!squash

**Pull request merged**